features/integration/abi.feature

```diff
@@ -12,6 +12,33 @@ Feature: ABI Interaction
     And I wait for the transaction to be confirmed.
     And I remember the new application ID.
 
+  Scenario Outline: Method call exceptions
```

Comment:
Also, since this scenario only tests for the incorrect number of arguments, I'd recommend renaming it to be more specific. The reason being this doesn't test every type of error that could happen, e.g. passing the wrong type is something not tested, and I think that would be difficult to test with the current steps.